### PR TITLE
Fix #38 mergin registration twice

### LINKED ISSUE #38: Two pages about Mergin registration
One page should be enough
- https://merginmaps.com/docs/howto/manage/web/register/
- https://merginmaps.com/docs/howto/sign-up-to-mergin-maps/

### PR DESCRIPTION
**Pull request by alex-cit**
- merged info from https://merginmaps.com/docs/howto/manage/web/register/ (deleted) and https://merginmaps.com/docs/howto/sign-up-to-mergin-maps/
- deleted page added to redirects

**There is an issue with headings with components (e.g. ## From <MainDomainName /> ) in table of contents (toc reference is #from-maindomainname ,  heading reference is  #from )

### CODE REVIEW

File: src/tutorials/opening-surveyed-data-on-your-computer/index.md [hunk 1]
```diff
@@ -2,11 +2,15 @@
 
 [[toc]]
 
-In the last tutorial you learnt how to FIXME.
+In the last tutorial you learnt how to capture field data using <MobileAppName />.
 
-In this tutorial you'll learn how to:
+In this tutorial you'll learn how to transfer data from mobile device to computer in seconds using QGIS, a free and open source desktop GIS package. <MobileAppName /> is based on QGIS, which means it's able to visualise and edit data in the same way QGIS can. This offers us great flexibility which we'll start to see in a moment.
```
Comment (PeterPetrik) [inline src/tutorials/opening-surveyed-data-on-your-computer/index.md:7]: I think we already have these changes in another PR. so hopefully we do not have conflicts during merging. In general if you work on the same part of the docs, feel free to have one bigger PR, where you mention multiple smaller tickets.

**Comment by PeterPetrik** [comment]
I think we should just remove the Vue component for the "mergnimaps name", input name etc...

File: src/howto/sign-up-to-mergin-maps/index.md [hunk 2]
```diff
@@ -11,12 +16,18 @@
...
+check your SPAM folder if the confirmation email does not appear in your inbox after 5 minutes.
+:::
+
+   You can now [skip to the Next steps](./index.md#next-steps).
 
 
 ## From <MobileAppName />
 
-If you already have [installed <MobileAppName /> (TODO: write this page)](../install-mergin-maps-mobile/index.md) installed then you can sign up from there.
+If you already have installed <MobileAppName /> installed then you can sign up from there.
```
[the remark this answers is not in the document]
Comment (alex-cit) [reply src/howto/sign-up-to-mergin-maps/index.md]: Fixed. There is no article about how to install Input App (the previous link went to 404). I added instructions how to download the app. I don't feel like this requires a separate article about installing the app - do you?

[the remark this answers is not in the document]
Comment (PeterPetrik) [reply src/howto/sign-up-to-mergin-maps/index.md]: nope, itt is fine

**Pull request merged**